guides/micronaut-openapi-generator-server/java/src/test/java/example/micronaut/library/controller/BooksControllerTest.java

```diff
@@ -0,0 +1,96 @@
+package example.micronaut.library.controller;
+
+import example.micronaut.library.model.BookAvailability;
```

Comment:
I had to manually add an import for `BookAvailability`, it wasn't in the generated code - is this a bug?